apis/metal3.io/v1alpha1/baremetalhost_validation_test.go

```diff
@@ -511,6 +602,94 @@ func TestValidateCreate(t *testing.T) {
 			oldBMH:    nil,
 			wantedErr: "Image URL test1 is an invalid URL",
 		},
+		{
+			name: "validStatusAnnotation",
+			newBMH: &BareMetalHost{
+				TypeMeta: tm, ObjectMeta: om_sa_valid,
```

Comment:
I wonder if it's useful to have these variables. This way, the test cases are separated from the actually tested configured. I'd just inline the content of ObjectMeta everywhere.

Reply:
Thanks, that makes sense, I have moved the content inline.